IBPSA/Fluid/Chillers/BaseClasses/InnerCycle_Chiller.mo

```diff
@@ -0,0 +1,209 @@
+within IBPSA.Fluid.Chillers.BaseClasses;
+model InnerCycle_Chiller "Blackbox model of refrigerant cycle of a chiller"
+  extends IBPSA.Fluid.HeatPumps.BaseClasses.PartialInnerCycle;
+  // Setting all values to zero avoids errors when checking this model.
+  // The values are correctly propagated by the heat pump / chiller model anyway
+  replaceable model BlaBoxChiCooling =
+      IBPSA.Fluid.Chillers.BlackBoxData.BaseClasses.PartialChillerBlackBox (
+       QUse_flow_nominal=0,
+       QUseBlackBox_flow_nominal=0,
+       scalingFactor=0,
+       TCon_nominal=0,
+       TEva_nominal=0,
+       dTCon_nominal=0,
+       dTEva_nominal=0,
+       primaryOperation=true,
+       mCon_flow_nominal=0,
+       mEva_flow_nominal=0,
+       y_nominal=0)
+    constrainedby
+    IBPSA.Fluid.Chillers.BlackBoxData.BaseClasses.PartialChillerBlackBox
+    "Replaceable model for performance data of a chiller in main operation mode"
+    annotation (choicesAllMatching=true);
+
+  replaceable model BlaBoxChiHeating =
+      IBPSA.Fluid.HeatPumps.BlackBoxData.BaseClasses.NoHeating(primaryOperation=false)
+    constrainedby
+    IBPSA.Fluid.HeatPumps.BlackBoxData.BaseClasses.PartialHeatPumpBlackBox
+    "Replaceable model for performance data of a chiller in reversible operation mode"
+    annotation (Dialog(enable=use_rev),choicesAllMatching=true);
+
+  BlaBoxChiCooling BlackBoxChillerCooling
+    annotation (Placement(transformation(extent={{7,20},{61,76}}, rotation=0)));
+  BlaBoxChiHeating BlackBoxChillerHeating if use_rev
+    annotation (Placement(
+        transformation(
+        extent={{-27,-28},{27,28}},
+        rotation=0,
+        origin={-34,48})));
+
+  Modelica.Blocks.Math.Gain gainEva(final k=-1)
+    "Negate QEva to match definition of heat flow direction" annotation (
+      Placement(transformation(
+        extent={{-4,-4},{4,4}},
+        rotation=180,
+        origin={-50,-6})));
+  Modelica.Blocks.Math.Gain gainCon(final k=-1) if use_rev
+    "Negate QCon to match definition of heat flow direction" annotation (
+      Placement(transformation(
+        extent={{-4,-4},{4,4}},
+        rotation=0,
+        origin={50,-20})));
+protected
+  Utilities.IO.Strings.StringPassThrough stringPassThrough;
```

Comment:
I changed this. Should the long paths always be used? Even if I'm in a sub-package and want to use the partial model in the base classes of that package?

Reply:
For instances, we can use the shorter name. But in hyperlinks use always the full name, e.g., <a href=\"modelica://IBPSA.Fluid.XYZ\">IBPSA.Fluid.XYZ</a>.
Otherwise, if classes are renamed, there is no way to check for proper renaming of the links. If one uses full names, the links will be renamed too by BuildingsPy. (For merging, it does not matter whether they are full names or partial names.)

Reply:
Thanks! I went through all models and fixed the links.